discord/abc.py

```diff
@@ -767,7 +767,7 @@ async def move(self, **kwargs):
 
         A rich interface to help move a channel relative to other channels.
 
-        If exact position movement is required, :meth:`edit` should be used instead.
+        If exact position movement is required, edit should be used instead.
```

Comment:
This cannot be linked to without:
a) documenting an abstract method `GuildChannel.edit`, or;
b) moving the `GuildChannel.move` `__doc__` to all GuildChannel.

Reply:
I've mentioned before that this is already intended and I don't really plan on changing it. Since it falls back to `span class=pre` then it conveys the information it needs. The linking is mainly a bonus.